Sources/AppStoreConnectCLI/Commands/TestFlight/Builds/ListBuildsCommand.swift

```diff
@@ -8,34 +8,56 @@ import Foundation
 struct ListBuildsCommand: CommonParsableCommand {
     static var configuration = CommandConfiguration(
         commandName: "list",
-        abstract: "Find and list builds for one app in App Store Connect.")
+        abstract: "Find and list builds for all apps in App Store Connect.")
 
     @OptionGroup()
     var common: CommonOptions
 
-    @Argument(help: "A bundle identifier that uniquely identifies an application.")
-    var bundleId: String
+    @Option(
+      parsing: .upToNextOption,
+      help: "An opaque resource ID that uniquely identifies the build"
+    )
+    var bundleId: [String]
```

Comment:
We have been prefixing all our filters with the `filter` keyword and specifying the value name as well

e.g.
```
    @Option(
        parsing: .upToNextOption,
        help: ArgumentHelp(
            "Filter by app bundle identifier. eg. com.example.App",
            discussion: "This option is mutually exclusive with --filter-app-ids.",
            valueName: "bundle-id"
        )
    ) var filterBundleIds: [String]
```

Also the help text provided here is actually the help text for an app id not a bundle id